bin/uid_entrypoint

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
please add description which explains what is purpose and need of this shell script; and why/when its helpful